dataland-frontend/src/components/forms/parts/kpiSelection/KPIfieldSet.vue

```diff
@@ -0,0 +1,166 @@
+<template>
```

Comment:
The name of this component should be `KpiFieldSet`, because we also write abbreviations in Pascal Case.

E.g. `EuTaxonomy...` even though "EU" is actually an abbreviation.

Reply:
It will be named DataPointForm after we marge main branch

Reply:
Reminder for myself:  Renaming to "DataPointForm" got lost during merge from main => retrieve it

Reply:
Has been retrieved.